src/DynamoCore/Graph/Nodes/NodeCategories.cs

```diff
@@ -420,8 +421,8 @@ internal static string MakeRelativePath(string basePath, string subjectPath)
             var documentUri = new Uri(basePath, UriKind.Absolute);
             var assemblyUri = new Uri(subjectPath, UriKind.Absolute);
 
-            var relativeUri = documentUri.MakeRelativeUri(assemblyUri);
-            var relativePath = relativeUri.OriginalString.Replace('/', '\\');
+            var relativeUri = documentUri.MakeRelativeUri(assemblyUri).OriginalString;
+            var relativePath = WebUtility.UrlDecode(relativeUri.Replace("+", "%2B")).Replace('/', '\\');
```

Comment:
this is part of the DynamoCore.dll... so I am pretty sure it will get built by mono.